### ISSUE TITLE
Running cosmopolitan binaries from Windows from normal windows command prompt messes with selecting commands

### ISSUE THREAD

**Issue by Keithcat1** [jart/cosmopolitan#199]
When using command prompt I like to run commands, then up-arrow, modify it and run it again. However running a Cosmopolitan-made binary on windows seems to break this. This is still happening with Cosmopolitan 1.0.
I thought that renaming hello.com to hello.exe would fix this, but it didn't seem to.

**Comment by pkulchenko** [comment]
I run redbean on windows (8 and 10) and can't reproduce this issue. I also modify the command and run it again, but haven't observed any issues occurring. Can you provide more details, exact command, and may be a screenshot to show what's happening?

**Comment by f-carraro** [comment]
It looks like the same issue I'm having on Windows 10. 
Open hello.com in cmd.exe, the program displays the text and exits correctly but after that, when trying to type the arrow keys, the command prompt returns: ^[[B^[[D^[[A^[[C.

Can you reproduce this?

**Comment by pkulchenko** [comment]
No, I can't reproduce this. Just to be clear, this is what I did:

1. Launch `cmd`
2. `cd cosmopolitan/o/rel/examples`
3. Launch `hello.com`
4. "hello world" is printed
5. Using arrow keys shows "hello.com" (when up/down used or char-by-char when left-right are used). I don't get anything else wrong. This is on Win10. The code is compiled in WSL2.

**Comment by f-carraro** [comment]
@pkulchenko 

Could you please compile the following program and see if the arrow keys are behaving normally for you in the scanf step in Win10?

```
int main() {
   char str[100];
   printf("Enter a value :");
   scanf("%s", str);
   printf("\nYou entered: %s", str );
   return 0;
}
```

**Comment by pkulchenko** [comment]
Yes, I can see the issue when arrow keys are used. Also, `Enter a value:` shows up for me *after* the value is entered, but it's probably because the output is not flushed.

**Comment by jart** [comment]
What's likely happening is your program is crashing. If the code in `exit()` doesn't get called then on Windows we can't take CMD.EXE out of virtual terminal mode.

https://github.com/jart/cosmopolitan/blob/053ee714bd155776961eb508d3ee463cd0ecae97/libc/runtime/exit.c#L40-L55

What I'd recommend trying is put the following at the start of your main function:

```c
showcrashreports();
```

That way Cosmopolitan will try to print a backtrace if your program segfaults. It works best if the .com.dbg binary is present in the same folder.

**Comment by f-carraro** [comment]
Actually the first issue have been already solved in #60, I was mistankanly running an earlier version of cosmopolitan for testintg if the second (scanf) was a regression.

The flush question has already been discussed in #181 and I'm ok with that. 

The one that remains is the one that @pkulchenko was able to reproduce with the scanf and arrow keys bisbehaving. I tried it in cosmopolitan 0.2 and 1.0, using both cmd and powershell and it always happened.
